quarto/Group_10_Analysis.qmd

```diff
@@ -0,0 +1,188 @@
+---
+title: "Analysis of IMDB Rating with GLM"
+format: html
+editor: visual
+---
+
+# Introduction {#sec-intro}
+
+Given the properties of films, predict whether the rating will be greater than 7 or not.
+
+```{r}
+#| echo: true
+#| warning: false
+
+library(tidyverse)
+library(gt)
+
+df <- read_csv("../data/dataset10.csv")
+
+df |> 
+  slice_head(n=5) |>
+  gt() |>
+  cols_label(
+    film_id = html("Film ID"),
+    year = html("Year"),
+    length = html("Length"),
+    budget = html("Budget"),
+    votes = html("Votes"),
+    genre = html("Genre"),
+    rating = html("Rating")
+  )
+```
+
+Description:
+
+-   Film ID: The unique identifier for the film
+-   Year: Year of release of the film in cinemas
+-   Length: Duration (in minutes)
+-   budget: Budget for the films production (in \$1000000s)
+-   votes: Number of positive votes received by viewers
+-   genre: Genre of the film
+-   rating: IMDB rating from 0-10
+
+# Data Wrangling
+
+## Missing value
+
+```{r}
```

Comment:
From @M-MWW: Calculate the number of NA, if the proportion is not large, we can remove it, if the number is large, maybe we need to fill with the mean value.

Reply:
yes,calculate the number of NA and visualize it